pkg/new-ui/v1beta1/frontend/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
Do we need this editor config since we have prettier ?

Reply:
Good question. I was using vim but hadn't set any plugin to work with `editorconfig` files https://editorconfig.org/#download, so 99% if there was an error I wouldn't have noticed it with my editor. In my case only prettier was formatting my files.

I'm not aware of the best practices here wrt having both a `prettierrc` file and an `editorconfig` one, or if they are not supposed to work together at all. We can do as you propose

Reply:
I think Prettier should be enough, so we can remove this config.

Reply:
pushed a commit that removes this file